services/cloudfront/src/main/java/software/amazon/awssdk/services/cloudfront/CloudFrontUtilities.java

```diff
@@ -233,6 +237,7 @@ public SignedUrl getSignedUrlWithCustomPolicy(Consumer<CustomSignerRequest.Build
      *     Path keyFile = myKeyFile;
      *     Instant activeDate = Instant.now().plus(Duration.ofDays(2));
      *     String ipRange = "192.168.0.1/24";
+     *     String policyResource = resourceUrl + "*";
```

Comment:
The '+' in the Java context could be misinterpreted as string concatenation. 
Can we just say 
String policyResource = resourceUrl(by default) or something like this which is not ambiguous